homeassistant/components/webostv/media_player.py

```diff
@@ -450,6 +451,9 @@ async def async_button(self, button):
         await self._client.button(button)
 
     @cmd
-    async def async_command(self, command):
+    async def async_command(self, command, **kwargs):
         """Send a command."""
-        await self._client.request(command)
+        argument = kwargs.get("argument")
+        if argument is not None:
+            argument = json.loads(argument)
+        await self._client.request(command, payload=argument)
```

Comment:
how does API handle payload=None ?

Reply:
It is valid for the API, basically it will just send a request to the endpoint without any arguments.